plugins/node/opentelemetry-instrumentation-aws-sdk/package.json

```diff
@@ -0,0 +1,80 @@
+{
+    "name": "@opentelemetry/instrumentation-aws-sdk",
+    "version": "0.25.0",
+    "description": "open telemetry automatic instrumentation for the `aws-sdk` package",
+    "keywords": [
+        "aws",
+        "opentelemetry",
+        "aws-sdk"
+    ],
+    "homepage": "https://github.com/open-telemetry/opentelemetry-js-contrib#readme",
+    "license": "Apache-2.0",
+    "author": "OpenTelemetry Authors",
+    "bugs": {
+        "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+    },
+    "main": "build/src/index.js",
+    "types": "build/src/index.d.ts",
+    "files": [
+        "build/src/**/*.js",
+        "build/src/**/*.js.map",
+        "build/src/**/*.d.ts",
+        "doc",
+        "LICENSE",
+        "README.md"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": "open-telemetry/opentelemetry-js-contrib",
+    "scripts": {
+        "clean": "rimraf build/*",
+        "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+        "compile": "npm run version:update && tsc -p .",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "precompile": "tsc --version && lerna run version --scope $(npm pkg get name) --include-filtered-dependencies",
+        "prewatch": "npm run precompile",
+        "prepare": "npm run compile",
+        "tdd": "npm run test -- --watch-extensions ts --watch",
+        "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/*.test.ts'",
+        "test-all-versions": "tav",
+        "version:update": "node ../../../scripts/version-update.js",
+        "watch": "tsc -w"
+    },
+    "peerDependencies": {
+        "@opentelemetry/api": "^1.0.1"
+    },
+    "dependencies": {
+        "@opentelemetry/core": "^0.25.0",
+        "@opentelemetry/instrumentation": "^0.25.0",
+        "@opentelemetry/semantic-conventions": "^0.25.0",
+        "opentelemetry-propagation-utils": "^0.24.0"
```

Comment:
Should we move this package also?

Reply:
Yes you are right, I need to move it as well.